sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/Metrics/GetMetricFeedbacksOptions.cs

```diff
@@ -25,9 +24,15 @@ public GetMetricFeedbackOptions(string metricId)
         /// <summary> The metric Id used to filter the feedbacks. </summary>
         public Guid MetricId { get; }
 
+        /// <summary> The dimension filter. </summary>
+        internal FeedbackDimensionFilter DimensionFilter { get; set; }
 
         /// <summary> The dimension filter. </summary>
-        public FeedbackDimensionFilter DimensionFilter { get; set; }
+        public DimensionKey Filter
+        {
+            get => new DimensionKey(DimensionFilter.Dimension);
```

Comment:
Can we have something like we do in `EmailHook.EmailsToAlert`? `Filter` would have simple getter and setter:
```cs
public DimensionKey Filter { get; set; }
```

And the internal `DimensionFilter` would be the one created during the get operation:
```cs
internal FeedbackDimensionFilter DimensionFilter => new FeedbackDimensionFilter(...);
```
We would need to update the constructors as well.

I think that creating a new `DimensionKey` when we call `get` can be a bit non-intuitive, since users would expect the same reference to be returned every time.